torch/testing/_internal/common_methods_invocations.py

```diff
@@ -623,6 +623,15 @@ def __init__(self,
                  assert_jit_shape_analysis=False,  # assert that jit shape analysis fully propagates shape
                  # the following metadata relates to ExpandedWeights support and is checked in test_expanded_weights.py
                  supports_expanded_weight=False,
+                 # FIXME: this should be changed to default to True
+                 # when all lambdas from torch/overrides have been ported to OpInfos
+                 # in order to throw errors that alert the user to fill in override_lambda
+                 supports_overrides=False,  # whether the op supports overriding
```

Comment:
"Overriding" is a very general term -- what kind of overriding?

Reply:
Is this metadata going to stick around post-PR simplification (per offline discussion)?

Reply:
By metadata do you mean the `supports_overrides` flag? Perhaps this could be deleted, as the initial error message from test/test_overrides should be able to throw an error.

Reply:
Yes, sorry, the attribute.